rev-connect-api/src/main/java/com/rev_connect_api/models/User.java

```diff
@@ -112,6 +103,14 @@ public void setBusiness(boolean isBusiness) {
         this.isBusiness = isBusiness;
     }
 
+    public String getUserPwd() {
+        return userPwd;
+    }
+
+    public void setUserPwd(String userPwd) {
+        this.userPwd = userPwd;
+    }
+
     @Override
```

Comment:
These nested `if` statements are tough to read and not very idiomatic. A more concise way is to return the comparison of the fields directly. This might be easier to read and maintain: 
```suggestion
@Override
public boolean equals(Object obj) {
    if (this == obj) return true;
    if (obj == null || getClass() != obj.getClass()) return false;

    User other = (User) obj;

    return isBusiness == other.isBusiness &&
           (userId != null ? userId.equals(other.userId) : other.userId == null) &&
           (username != null ? username.equals(other.username) : other.username == null) &&
           (userPwd != null ? userPwd.equals(other.userPwd) : other.userPwd == null) &&
           (email != null ? email.equals(other.email) : other.email == null) &&
           (firstName != null ? firstName.equals(other.firstName) : other.firstName == null) &&
           (lastName != null ? lastName.equals(other.lastName) : other.lastName == null);
}
```